asm/stage_intro.s

```diff
@@ -151,18 +151,18 @@ gUnknown_080D6FF7:
     @ Colors [NUM_CHARACTERS]
     .global sZoneLoadingCharacterColors
 sZoneLoadingCharacterColors:
-    .2byte 0x7C00       @ RGB( 0,  0, 31) // Sonic
-    .2byte 0x025F       @ RGB(31, 18,  0) // Cream
-    .2byte 0x037F       @ RGB(31, 27,  0) // Tails
-    .2byte 0x007F       @ RGB(31,  3,  0) // Knuckles
-    .2byte 0x563F       @ RGB(31, 17, 21) // Amy
+    RGB  0  0 31   @; Sonic
```

Comment:
This macro is cool but seems not needed in asm as this array is defined in C

Reply:
But it can't hurt, either. ^^
Just thought it'd be good to have around!